Mathlib/LinearAlgebra/Matrix/Trace.lean

```diff
@@ -239,12 +239,10 @@ variable [Fintype n] [AddCommMonoid α] (i j : n) (c : α)
 
 @[simp]
 theorem trace_zero (h : j ≠ i) : trace (stdBasisMatrix i j c) = 0 := by
-  -- Porting note: added `-diag_apply`
```

Comment:
I'm inclined to keep this porting note since it suggests the simp set is not working well here.